package.json

```diff
@@ -18,9 +18,14 @@
     "chrome-types": "^0.1.263",
     "esbuild": "^0.20.0",
     "esbuild-plugin-copy": "^2.1.1",
-    "esbuild-sass-plugin": "^3.0.0"
+    "esbuild-sass-plugin": "^3.0.0",
+    "postcss": "^8.4.35",
+    "postcss-preset-mantine": "^1.13.0",
+    "postcss-simple-vars": "^7.0.1"
```

Comment:
Understand how exactly `mantine` uses `postcss-...` stuff. See if we can get away without depending on these.

They don't affect bundle size, but we're still pulling a bunch of things into node modules o.O

Reply:
^ okay looks like they are not _required_, just recommended; and we don't currently use them. Hence removed.